Makefile

```diff
@@ -331,7 +331,7 @@ test:
 	skipper make $(SKIPPER_PARAMS) _test
 
 _test: $(REPORTS) _test_setup
-	JUNIT_REPORT_DIR=$(REPORTS) python3 ${DEBUG_FLAGS} -m pytest $(or ${TEST},src/tests) -k $(or ${TEST_FUNC},'') -m $(or ${TEST_MARKER},'') --verbose -s --junit-xml=$(PYTEST_JUNIT_FILE)
+	JUNIT_REPORT_DIR=$(REPORTS) python3 ${DEBUG_FLAGS} -m pytest --error-for-skips $(or ${TEST},src/tests) -k $(or ${TEST_FUNC},'') -m $(or ${TEST_MARKER},'') --verbose -s --junit-xml=$(PYTEST_JUNIT_FILE)
```

Comment:
hmm, it's problematic no?
because ``-k/-m`` will also be considered as a skip no?

Reply:
Oh.. I get it now, I don't think it's considered as skip, I think that skip it's when the test is "running" but it's skipped, the - k for example is running only the specified tests (starts with TEST_FUNC), so when running a specific test (e.g. test_install) it will run just those tests, and if one of those tests is marked as skipped than it will fail

Reply:
I also tested it locally